app/views/shared/check_answers/_assets.html.erb

```diff
@@ -1,13 +1,18 @@
 <% read_only = false unless local_assigns.key?(:read_only) %>
 <% online_savings_accounts = @legal_aid_application.online_savings_accounts_balance %>
 <% online_current_accounts = @legal_aid_application.online_current_accounts_balance %>
+<% individual = if @legal_aid_application.applicant.has_partner_with_no_contrary_interest?
+                  "your client or their partner"
+                else
+                  "your client"
+                end %>
```

Comment:
Should we add these content to our locales file?

Reply:
Probably yes, but this pattern is used throughout the new partner pages.  
I'd suggest a tidy up ticket to extract them all to a shared local to avoid the repetition :+1: